proxies/sandbox/apiproxy/policies/AssignMessage.AddAsidHeader.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Pretty sure all these policies are unused in the Sandbox. I'll make a ticket to refactor: APM-565